app/models/game.rb

```diff
@@ -3,4 +3,124 @@ class Game < ApplicationRecord
   has_many :pieces
 
   enum game_status: { in_progress: 0, checkmate: 1, stalemate: 2 }
+
+  def populate_board!
+    # this should create all 32 pieces with their initial X/Y coordinates.
+    
+    # Build white pieces
+
+    (0..7).each do |i|
+      Pawn.create(
+        row_coordinate: 6,
+        column_coordinate: i,
+        color: "white",
+        )
```

Comment:
We had put game.id for each piece, but took it out because we thought that they would automatically be given the game id  😂 😹